test/orders/service/OrdersService.spec.ts

```diff
@@ -602,10 +626,18 @@ Code: 15 Message: Requested business cannot be found.`;
         expect(actualOrder.customer!.shippingAddress!.address1).to.equal(newAddress1);
         expect(actualOrder.customer!.shippingAddress!.city).to.equal(newCity);
         expect(actualOrder.customer!.shippingAddress!.province).to.equal(newProvince);
+        expect(actualOrder.lineItems[0].taxes!.length).to.equal(2);
+        expect(actualOrder.lineItems[0].taxes![0].title).to.equal('PA State Tax');
+        expect(actualOrder.lineItems[0].taxes![0].price).to.equal(9.99);
+        expect(actualOrder.lineItems[0].taxes![0].rate).to.equal(0.06);
+        expect(actualOrder.lineItems[0].taxes![1].title).to.equal('NV State Tax');
+        expect(actualOrder.lineItems[0].taxes![1].price).to.equal(1.00);
+        expect(actualOrder.lineItems[0].taxes![1].rate).to.be.undefined;
       });
     });
 
     it('And valid OrderCreateRequest with an actionId when creating an order, Then return created order', () => {
+      console.log(JSON.stringify(singleOrder));
```

Comment:
@ryankazokas Forgot a debugging console.log?

Reply:
Yes, thank you. Good catch.